content/en/blog/2022-01-31-january-update/index.md

```diff
@@ -0,0 +1,538 @@
+---
+author: dholbach
+date: 2022-01-31 9:30:00+00:00
+title: January 2022 Update
+description: New Flux and Flagger releases bring more security, terraform-controller team wants feedback, Flux articles and docs, upcoming Flux events helping you get started and more.
+url: /blog/2022/01/january-update/
+resources:
+- src: "**.png"
+  title: "Image #:counter"
```

Comment:
For some reason the preview image is not shown on https://deploy-preview-730--fluxcd.netlify.app/blog/ - will figure it out.